tests/phpunit/tests/block-bindings/register.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Tests for Block Bindings API helper functions.
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ * @since 6.5.0
+ *
+ * @group blocks
+ * @group block-bindings
+ */
+class Tests_Block_Bindings_Register extends WP_UnitTestCase {
+
+	const TEST_SOURCE_NAME       = 'test/source';
+	const TEST_SOURCE_PROPERTIES = array(
+		'label' => 'Test source',
+	);
+
+	/**
+	 * Tear down after each test.
+	 *
+	 * @since 6.5.0
+	 */
+	public function tear_down() {
+		foreach ( get_all_registered_block_bindings_sources() as $source_name => $source_properties ) {
+			if ( str_starts_with( $source_name, 'test/' ) ) {
+				unregister_block_bindings_source( $source_name );
+			}
+			unregister_block_bindings_source( $source_name );
```

Comment:
I think there is a mistake here. Do we want to unregister all sources or only the ones whose name starts with `test/` ?

Reply:
Yes, this line needs to get removed. It should unregister only these prefixed with `test/` 👍🏻

Reply:
Oh, I see. I removed the 

```
if ( str_starts_with( $source_name, 'test/' ) ) {
  unregister_block_bindings_source( $source_name );
}
```

in https://github.com/WordPress/wordpress-develop/pull/5888/commits/3f242d3caf3e8b2a0439945a290e9554722b07e8#diff-c61c6a9202e942d0484d2a18d9b4601815d15a6aef2a659b9dc1e0454db26ea9L26-L28 because a unit test was failing, so I'll change it back.